app/queries/gobierto_data/datasets/creation_statements.rb

```diff
@@ -57,7 +57,7 @@ def complete_schema_with_defaults(source_file, schema_definition)
         default_schema = inspect_csv_schema(source_file, csv_separator: @csv_separator)
         schema_definition = (schema_definition || {}).deep_symbolize_keys
         default_schema.map do |default_column, default_value|
-          schema_definition.find { |_, value| value[:original_name] == default_value[:original_name] } || [default_column, default_value]
+          schema_definition.find { |_, value| value[:original_name].casecmp?(default_value[:original_name]) } || [default_column, default_value]
```

Comment:
Why the original_name is mandatory? I think it should be optional if the name doesn't change

Reply:
Changed, now this attribute can be omitted